system/web/services/RoutingService.cfc

```diff
@@ -156,14 +156,13 @@ component extends="coldbox.system.web.services.BaseService" accessors="true"{
 		return this;
 	}
 
-	/****************************************************************************************************************************/
-	/* 													INTERCEPTION EVENT 														*/
-	/****************************************************************************************************************************/
-
 	/**
 	 * This is the route dispatcher called upon the request is captured.
 	 */
-	public void function onRequestCapture( event, interceptData, rc, prc, buffer ){
+	public void function requestCapture( event ) {
```

Comment:
Can you make the argument required please and add it to the cfc doc with what we are receiving please.